### PR TITLE
Cocoapods.md

### PR DESCRIPTION
**Pull request by Fatme**

### CODE REVIEW

**Comment by Fatme** [comment]
@ikoevska

File: CocoaPods.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
+# Using CocoaPods
+
+When you develop for iOS, you can quickly add third party libraries to your NativeScript projects via [CocoaPods](https://cocoapods.org/), a dependency manager for Swift and Objective-C Cocoa projects.
```
Comment (ikoevska) [inline CocoaPods.md]: third-party

File: CocoaPods.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+## Table of Contents
```
Comment (ikoevska) [inline CocoaPods.md]: You can remove this line - generally, we don't include it.

File: CocoaPods.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+ * [Install CocoaPods](#install-cocoapods)
+ * [Create CLI Project](#create-cli-project)
+ * [Create NativeScript plugin with Podfile](#create-nativescript-plugin-with-podfile)
+ * [Build the Project](#build-the-project)
+
+## Install CocoaPods
+ You need to install CocoaPods. If you haven't yet, you can do so by:
```
Comment (ikoevska) [inline CocoaPods.md]: by running

File: CocoaPods.md [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+```
+$ sudo gem install cocoapods
+```
+> **NOTE:** All operations and code in this article are verified against CocoaPods 0.38.2.
+
+You can check your current version using:
```
Comment (ikoevska) [inline CocoaPods.md]: To check your current version, run the following command.

File: CocoaPods.md [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+```
+$ pod --version
+```
+
+To update CocoaPods just install the gem again with:
```
Comment (ikoevska) [inline CocoaPods.md]: To update CocoaPods, just run the installation command again.

File: CocoaPods.md [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+```
+sudo gem install cocoapods
+```
+
+## Create CLI Project
+To start, create a project, add the iOS platform.
```
Comment (ikoevska) [inline CocoaPods.md]: create a project and add

File: CocoaPods.md [hunk 7]
```diff
@@ -0,0 +1,83 @@
...
+```bash
```
Comment (ikoevska) [inline CocoaPods.md]: You might need an empty line before this one for proper rendering. (This applies to all code blocks.)

File: CocoaPods.md [hunk 8]
```diff
@@ -0,0 +1,83 @@
...
+$ tns create MYCocoaPods
+$ cd MYCocoaPods
+$ tns platform add ios
+```
+
+## Create NativeScript plugin with Podfile
+
+For more information see [NativeScript plugins](#PLUGINS.md)
```
Comment (ikoevska) [inline CocoaPods.md]: For more information about working with NativeScript plugins, click here. (here being the link)

You need to state what this more information is about as right now the context is not so obvious.

File: CocoaPods.md [hunk 9]
```diff
@@ -0,0 +1,83 @@
...
+
+```
+cd ..
+mkdir my-plugin
+cd my-plugin
+touch package.json
+```
+
+Create a package.json file with the following content:
```
Comment (ikoevska) [inline CocoaPods.md]: Didn't the `touch package.json` command create the package.json file? If so, make this "In the newly created package.json file, insert the following syntax."

File: CocoaPods.md [hunk 10]
```diff
@@ -0,0 +1,83 @@
...
+  "version": "0.0.1",
+  "nativescript": {
+    "platforms": {
+      "ios": "1.3.0"
+    }
+  }
+}
+```
+
+Create a [Podfile](https://guides.cocoapods.org/syntax/podfile.html) and move it to platforms/ios folder.
```
Comment (ikoevska) [inline CocoaPods.md]: Should this be an empty Podfile?  This part needs to be more specific.

File: CocoaPods.md [hunk 11]
```diff
@@ -0,0 +1,83 @@
...
+
+```
+my-plugin/
+├── package.json
+└── platforms/
+    └── ios/
+        └── Podfile
+```
+
+Next install the plugin:
```
Comment (ikoevska) [inline CocoaPods.md]: Comma after next

File: CocoaPods.md [hunk 12]
```diff
@@ -0,0 +1,83 @@
...
+```
+
+## Build the project
+```
+tns build ios
+```
+
+This modifies the `MYCocoaPods.xcodeproj` and creates a workspace with the same name.
+
+> **IMPORTANT:** You will no longer be able to run the `xcodeproj` alone. {N} CLI will build the newly created workspace and produce correct .ipa.
```
Comment (ikoevska) [inline CocoaPods.md]: The NativeScript CLI

produce the correct .ipa

**Comment by Fatme** [comment]
:+1:  from @ikoevska

**Pull request merged**